GVRf/Sample/gvr-sample/src/org/gearvrf/sample/SampleCubeScript.java

```diff
@@ -160,23 +153,20 @@ public void onInit(GVRContext gvrContext) {
     @Override
     public void onStep() {
         FPSCounter.tick();
-        mFrontBoardObject.getRenderData().getMaterial().setOpacity(1.0f);
-        mFrontBoardObject2.getRenderData().getMaterial().setOpacity(1.0f);
-        mFrontBoardObject3.getRenderData().getMaterial().setOpacity(1.0f);
+        mFrontFace.getRenderData().getMaterial().setOpacity(1.0f);
```

Comment:
Why are you setting opacity to constant values in `onStep()`? Shouldn't this happen once, in`onInit()`?